rancher/installing-rancher/installing-server/multi-nodes/index.md

```diff
@@ -71,11 +75,18 @@ _Available as of v1.0.1_
 5. Click on **Generate Config Script**. 
 6. Download the script and save it locally. 
 
-> **Note:** After the configuration script is saved, you can stop and remove the script generating Rancher server container. This will allow you to reuse that node for the HA setup.  
+> **Note:** After the configuration script is saved, you should stop and remove the script generating Rancher server container. You should remove this container before running the HA script to add the same node to your HA setup.  
 
 ### Launching Rancher in HA
 
-1. For each node that you want in HA, use the startup script to launch Rancher server. The script will start a Rancher server container that connects to the same external MySQL database created earlier.  
+1. For each node that you want in HA, use the startup script to launch Rancher server on all nodes. The script will start a Rancher server container that connects to the same external MySQL database created earlier. 
+
+   > **Note:** If you are reusing the node that created the HA script, please stop and remove the script generating Rancher server container before running the script.
```

Comment:
It might be safe to assume that most everyone is reusing the 1st server in their HA setup. We could simplify this statement:

Please stop rancher-server container after you generate the rancher-ha.sh launch script. Failure to do so will cause port conflicts.